pkg/sql/plan/function/builtin/debug/cmd_flush.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2021 - 2022 Matrix Origin
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package debug
+
+import (
+	"github.com/matrixorigin/matrixone/pkg/common/moerr"
+	"github.com/matrixorigin/matrixone/pkg/container/types"
+	pb "github.com/matrixorigin/matrixone/pkg/pb/debug"
+	"github.com/matrixorigin/matrixone/pkg/vm/engine/tae/db"
+	"github.com/matrixorigin/matrixone/pkg/vm/process"
+	"strings"
+)
+
+func handleFlush() handleFunc {
+	return getDNHandlerFunc(
+		pb.CmdMethod_Flush,
+		func(_ string) ([]uint64, error) {
+			return nil, nil
+		},
+		func(dnShardID uint64, parameter string, proc *process.Process) []byte {
+			// parameter should be "DbName@TableName"
+			parameters := strings.Split(parameter, "@")
+			payload, err := types.Encode(db.FlushTable{
+				DatabaseName: parameters[0],
+				TableName:    parameters[1],
+				AccessInfo: db.AccessInfo{
+					AccountID: proc.SessionInfo.AccountId,
+					UserID:    proc.SessionInfo.UserId,
+					RoleID:    proc.SessionInfo.RoleId,
+				},
+			})
+			if err != nil {
+				panic(any(moerr.NewInternalError("payload encode err")))
```

Comment:
you can't panic here, because the parameters are input outside the mo system and cannot be guaranteed to be correct. If the parsing fails, an error should be returned to tell the user